pkg/python/parse_test.go

```diff
@@ -0,0 +1,54 @@
+package python
+
+import (
+	"os"
+	"path"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func TestParse(t *testing.T) {
+	vectors := []struct {
+		file string // Test input file
+		want []types.Library
+	}{
+		{
+			file: "testdata/requirements_flask.txt",
+			want: requirementsFlask,
```

Comment:
Please add more tests. `requirements.txt` may include requirements without version specifiers, other requirements files, etc.
https://pip.pypa.io/en/latest/cli/pip_install/#requirements-file-format

Reply:
Also, the version may have spaces after `==` as follows.
```
docopt == 0.6.1
```

Reply:
Added whitespace stripping and test coverage for this. Let me know if more is required.